pkg/views/workspace/selection/workspace.go

```diff
@@ -39,6 +46,18 @@ func generateWorkspaceList(workspaces []apiclient.WorkspaceDTO, isMultipleSelect
 
 		stateLabel := views.GetStateLabel(workspace.State.Name)
 
+		isDisabled := false
```

Comment:
We should definitely sort workspaces by "disablement". Those that are disabled should go on the bottom.

This can also make it easier to not even allow the user to move their cursor over the disabled items.
If the user tries to "move down" into the disabled workspaces, we can just prevent that cursor action completely.
We had this functionality somewhere but I can't remember where 😅 (we might've removed it at some point)

@idagelic thoughts on this?

Reply:
We had this option when prompting for duplicate project repos / namespaces that only contain duplicate project repos back when we didnt support two of the same repository in a workspace. The approach with "disabling" is good, you can check it out here https://github.com/daytonaio/daytona/commit/5dee98ad9e5aea827503800c55957d70ab4ebf43

Reply:
https://github.com/user-attachments/assets/ce46b533-81db-43ca-8421-7efb4f7f09fb